script/glam/generate_glean_sql

```diff
@@ -66,7 +66,7 @@ function write_clients_daily_aggregates {
     tables=$(
         bq ls "$qualified" \
         | grep TABLE \
-        | grep -E 'schema_id:glean_ping_\d+' \
+        | grep -E 'schema_id:glean_ping_[0-9]\+' \
```

Comment:
`\+` doesn't work properly with `grep -E` for me locally on a Mac or in the Docker image.
```suggestion
        | grep -E 'schema_id:glean_ping_[0-9]+' \
```

Reply:
Thanks for catching this, I originally tested without the -E and it worked.